records_mover/db/vertica/records_export_options.py

```diff
@@ -44,14 +44,17 @@ def vertica_export_options(unhandled_hints: Set[str],
         cant_handle_hint(fail_if_cant_handle_hint, 'dateformat', hints)
     quiet_remove(unhandled_hints, 'dateformat')
 
-    if hints['datetimeformattz'] != 'YYYY-MM-DD HH:MI:SSOF':
+    if hints['datetimeformattz'] not in ['YYYY-MM-DD HH:MI:SSOF',
+                                         'YYYY-MM-DD HH24:MI:SSOF']:
         cant_handle_hint(fail_if_cant_handle_hint, 'datetimeformattz', hints)
     quiet_remove(unhandled_hints, 'datetimeformattz')
 
-    if hints['datetimeformat'] != 'YYYY-MM-DD HH:MI:SS':
+    if hints['datetimeformat'] not in ['YYYY-MM-DD HH:MI:SS',
+                                       'YYYY-MM-DD HH24:MI:SS']:
         cant_handle_hint(fail_if_cant_handle_hint, 'datetimeformat', hints)
     quiet_remove(unhandled_hints, 'datetimeformat')
 
+    # TODO: Add test that reproduces HH:MM:SS
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker